catt/controllers.py

```diff
@@ -26,6 +26,20 @@ def get_stream_info(video_url):
     return stream_info
 
 
+def get_chromecast(device_name):
+    devices = pychromecast.get_chromecasts()
+    if device_name:
```

Comment:
It doesn't distinguish though, does it? If you specify a device name it always says "Specified not found", otherwise "None found". If you specify a device name and it finds no devices, it will still say "Specified not found". Or do you mean in case you specified a device in the config and forgot about it? It could be useful then, I agree.